start.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+rm -rf wd_dumps
+
+git clone https://github.com/WikiAr/wd_dumps.git -b update
+
+cd wd_dumps
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)